ckanext/unaids/react/components/FileInputComponent/src/App.js

```diff
@@ -155,7 +153,8 @@ export default function App({
                 />
             );
         } else {
-            // e.g. [undefined, null, "file"].includes(uploadMode)
+            // essentially this is catchin any of [undefined, null, "file"]
+            // and anything else will also  be treated as this general case
```

Comment:
I think this comment creates more confusion then help. Either we actually care about value of uploadMode here (then you should put the condition in else if) or we don't care and any value is treated as general case.

If a developer would like to get all possible values of uploadMode then he should look for usage of setUploadMode/setHiddenInputs in the code. Such comment is extremely prone to become outdated and in sucha  case be even more confusing.